docs/kubeapps-on-bkpr.md

```diff
@@ -0,0 +1,122 @@
+# Installing Kubeapps on BKPR
+
+This guide documents the installation of [Kubeapps](https://kubeapps.com/) to your Kubernetes Production Runtime (BKPR) cluster. You will first install the [Helm](https://www.helm.sh/) package manager to the cluster, followed by the installation of Kubeapps using the [Kubeapps Helm chart](https://hub.kubeapps.com/charts/bitnami/kubeapps).
+
+![Kubeapps Application Catalog](images/kubeapps-app-catalog.png)
```

Comment:
Would it make sense to move this image to the final step, after the login screen?